bundle/config/mutator/resourcemutator/process_yaml_resources.go

```diff
@@ -0,0 +1,54 @@
+package resourcemutator
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databricks/cli/bundle"
+	"github.com/databricks/cli/libs/diag"
+	"github.com/databricks/cli/libs/dyn"
+)
+
+type processYamlResources struct{}
```

Comment:
> processYamlResources

I don't think we need to mention Yaml there. The important thing is that it's static resources defined in configuration. The actual encoding could easily be changed (e.g. we may add support for including json files and that would be a very local change).

So maybe processStaticResources?

Reply:
👍 Fixed